scripts/exclude-links.txt

```diff
@@ -0,0 +1,20 @@
+http://127.0.0.1:10000/devstoreaccount1;
+http://localhost:3000/
+https://$
+https://api.github.com/repos/$
+https://blog.$
+https://discuss.$
+https://dvc.org/some.link
```

Comment:
> https://dvc.org/some.link

Not seeing this one anywhere in the code 🙂

Reply:
feel free to remove it in the next regular stuff :)

Reply:
Will do, thanks.

Reply:
Ah might be a holdover from some of my local tests. Thought I'd cleaned all of them but must've missed this one

Reply:
p.s. addressing these small things in #978 🙂